tox.ini

```diff
@@ -1,9 +1,11 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
-envlist = py26,py27,py34
+envlist = py27,py34,py35
 
 [testenv]
+commands=nosetests
 deps=
    nose
    coverage
-commands=nosetests
+   -rrequirements.txt
```

Comment:
I believe you can accomplish the same thing by adding a `tests_require` section to the `setup.py`